whitepaper/chapters/consensus.tex

```diff
@@ -104,7 +104,7 @@ \subsection{Weighting Algorithm}
 Together, the transaction and node scores are called the \emph{activity score} because they are both dynamic and derived from an account's activity as opposed to its stake.
 The transaction score is weighted at 80\% and the node score at 20\%.
 Additionally, the combined score is scaled relative to an account's balance so that there is a dampening effect of activity on importance as stake increases
-\footnote{The activity score is rescaled after dampening so that it contributes the desired \nemsetting{network}{importanceActivityPercentage} to the importance calculation.}.
+\footnote{The activity score is rescaled after dampening so that it contributes to the desired \nemsetting{network}{importanceActivityPercentage} to the importance calculation.}.
```

Comment:
this changes meaning. it is trying to say that rescaling happens so that the activity score takes up the specified importanceActivityPercentage of score. Without rescaling, dampening would reduce the contribution of the activity score.